### ISSUE TITLE
Dynamic content blocked by SW

### ISSUE THREAD

**Issue by MESWEB** [mdn/sw-test#40]
I have dynamic content which one change something on website like number of sold items. I'm displaying number of orders on web page based on database connection. When number change I don't see this updated content and after second time reload page updated number is show. For example if customers see number 0 of orders and another customer order item then first one which one seen 0 at first visit next time he will see again 0 order even if real counter will be much higher than 0 because he get cached website from first visit. This Service Worker is maybe good for static context not for dynamical. Is there any way to disable caching DIV content?

**Comment by schalkneethling** [comment]
Hey @MESWEB! Thank you for opening this issue. We are combining some of our repositories to enable us to triage pull requests and issues more effectively. Therefore, this specific repository will be merged into the more prominent [dom-examples](https://github.com/mdn/dom-examples) repository.

Generally, for questions such as this, you would get a quicker response either via our Discourse instance(https://discourse.mozilla.org/c/mdn/236) or StackOverflow.
